pastehunter.py

```diff
@@ -230,6 +230,17 @@ def paste_scanner():
                     # Else use the rule name
                     else:
                         results.append(match.rule)
+
+                # Store additional fields for passing on to post processing
+                encoded_paste_data = raw_paste_data.encode('utf-8')
```

Comment:
Interesting. Why did you move this logic to be before the blacklist check/postprocessing? It shouldn't impact the shas if that was the intent here.

Reply:
In my case it was specifically because I wanted the rules that matched (`YaraRule`) to be available to the sandbox modules. It seemed to make the most sense to just move this entire block up before postprocessing so that all the final `paste_data` would be available to any post processing module.